TL improvement (#538)
07th-mod/higurashi-console-arcs: Update/omot_009.txt

//r梨花ちゃんから聞かされた、去年叔父夫婦と暮らしていたときの沙都子の生活は、酷いと一言では片付けられないほどのものだった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "梨花ちゃんから聞かされた、去年叔父夫婦と暮らしていたときの沙都子の生活は、酷いと一言では片付けられないほどのものだった。",

> 			NULL, "According to Rika-chan, Satoko's life with her uncle last year was beyond the word awful.", Line_Normal);

Reviewer: ```suggestion
			NULL, "According to Rika-chan, Satoko's life with her uncle last year was so far beyond awful that the word didn't cut it.", Line_Normal);
```

//r葛西さんは…とチラリと視線をやると、パフェには一切手をつけず、優雅にブラックコーヒーをすすっていた。…いつの間にコーヒーなんて頼んでいたんだ？
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "葛西さんは…とチラリと視線をやると、パフェには一切手をつけず、優雅にブラックコーヒーをすすっていた。…いつの間にコーヒーなんて頼んでいたんだ？",

> 			NULL, "What's up with Kasai-san...? I peeked at him. He was gracefully sipping his coffee, without touching his parfait at all. ...When did he even ask for that coffee?", Line_Normal);

Reviewer: ```suggestion
			NULL, "When I glanced at Kasai-san... he was gracefully sipping his coffee, without touching his parfait at all. ...When did he even ask for that coffee?", Line_Normal);
```

//r詩音の冗談にも、口元だけの微笑で全く動じたそぶりを感じられない。付き合いの長さもあるだろうけど、それ以上に男としての懐の深さと強さがうかがえた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "詩音の冗談にも、口元だけの微笑で全く動じたそぶりを感じられない。付き合いの長さもあるだろうけど、それ以上に男としての懐の深さと強さがうかがえた。",

> 			NULL, "Even after Shion's joke, I couldn't detect any reaction out of him except his lip with a slight smile. They seemed to have a long relationship together, but more than that I was able to see the depth and strength of him as a man.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Even after Shion's joke, I couldn't detect any reaction out of him except a slight smile on his lips. They seemed to have a long relationship together, but more than that I was able to see the depth and strength of him as a man.", Line_Normal);
```
